### DISCUSSION TITLE
Is there a MQTT command for limiting the outdoor unit capacity

### DISCUSSION THREAD

**Discussion by wurmtal868** [revk/xESP32-Faikout#203] [Q&A] [answered]
Having outdoor temperatures around 10°C, I'm experimenting with the demand control of the the Onecta App 
![Screenshot_20240126-124117](https://github.com/revk/ESP32-Faikin/assets/32965910/9ad8f3b0-13ec-4f76-85c1-3e09b61297a8)

Does the Faikin SW provide a MQTT command for getting and setting this power limit?
By setting this limit to a minimum I was able to run my out door unit contentiously at 10°C outside while keeping 20° C inside. Whether the limit was set to "off" or "auto", my outdoor unit was cycling on and off once every 5 minutes and consumed approximately 20-30% more energy.

**Comment by revk** [comment, marked as answer]
It does now (as of this week) have demand control settings, if your unit does that.

E.g. `command/Faikin/demand 75`

**Comment by wurmtal868** [comment]
After OTA I can see it in the status :-)
![image](https://github.com/revk/ESP32-Faikin/assets/32965910/3f0d3753-df44-46a0-8982-97773f7a499c)
